src/components/Button/Button.js

```diff
@@ -338,32 +372,38 @@ class Button extends React.Component<Props, State> {
 
     if (listItemButton || extraSmall || regularText) {
       return (
-        <ButtonTextRegular customTheme={customTheme} small={small} style={textStyle}>
+        <ButtonTextRegular theme={updatedTheme} customTheme={customTheme} small={small} style={textStyle}>
           {title}
         </ButtonTextRegular>
       );
     }
 
     return (
-      <ButtonText customTheme={customTheme} small={small} style={textStyle}>
+      <ButtonText theme={updatedTheme} customTheme={customTheme} small={small} style={textStyle}>
         {title}
       </ButtonText>
     );
   };
 
   render() {
-    const customTheme = getTheme(this.props);
+    // const customTheme = getTheme(this.props);
```

Comment:
```suggestion
```

Reply:
I swear I've deleted this line :)) Will remove it. Thanks!